app/views/posts/show.html.haml

```diff
@@ -16,16 +16,24 @@
 
 = render :partial => "single", :locals => { :post => @post, :subject => false, :hide_comments => true }
 
-- if can? :edit, @post or can? :destroy, @post
-  .post-actions
-    - if can? :edit, @post
-      = link_to 'Edit Post', edit_post_path(@post), :class => 'btn btn-default btn-xs'
-    - if can? :destroy, @post
-      = link_to 'Delete Post', @post, method: :delete, |
-        data: { confirm: 'Are you sure?' }, :class => 'btn btn-default btn-xs'
+- unless @post.crops.empty?
+  %h3 Crops mentioned in this post
+  - @post.crops.each do |c|
+    - unless c.photos.empty?
+      .col-md-2
+        = render :partial => 'crops/thumbnail', :locals => { :crop => c }
```

Comment:
I think we should show a thumbnail even if there are no photos. The way it is now, potentially the `h3` can be shown with no thumbnails underneath it.